projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```
@@ -118,6 +116,9 @@
     z-index: map.get(variables.$clr-layers, dropdown-menu);
 
     .datepicker-view-manager {
+      display: flex;
+      flex-direction: column;
+      row-gap: calc(2 * datepicker-variables.$clr-calendar-button-gap);
```

Whys `row-gap` is based on $clr-calendar-button-gap? Make sense to be regular token.
```suggestion
      row-gap: tokens.$cds-global-space-5;
```

---

Fixed in [948b80d](https://github.com/vmware-clarity/ng-clarity/pull/1706/commits/948b80d3d202cba561298fdb5a6636cfd8e84a57)